h2/src/main/org/h2/mvstore/Cursor.java

```diff
@@ -13,42 +13,76 @@
  * @param <K> the key type
  * @param <V> the value type
  */
-public class Cursor<K, V> implements Iterator<K> {
+public final class Cursor<K, V> implements Iterator<K> {
```

Comment:
Why is the class no final (I'm not a fan of using "final" if not needed, except for fields).

Reply:
see above for the reason why I am a fan of final classes